doc/systest_basics.rst

```diff
@@ -0,0 +1,288 @@
+  :class:`stile.sys_tests.BaseScatterPlotSysTest` (especially the method
+  :func:`scatterPlot <stile.sys_tests.BaseScatterPlotSysTest.scatterPlot>`) for more details.
+  Note that this type has a different call signature than
+  the other methods and that it lacks many of the convenience variables the other
+  ScatterPlots have, such as self.objects_list and self.required_quantities.
+
+Calls to child classes of :class:`stile.sys_tests.BaseScatterPlotSysTest`
+return :class:`matplotlib.figure.Figure` instances and can be saved directly as images, but
+a :func:`.plot` method is included (that just returns the figure passed to it) so the plotting
+interface is the same as other :class:`SysTests <stile.sys_tests.SysTest>`.
+
+You can turn the trendlines on and off with the kwarg ``reference_line='zero'`` (at :math:`y=0`),
+``reference_line='one-to-one'`` (at :math:`x=y`), or ``reference_line=function`` (to
+overplot the line given by ``function(x)``, which must return a 1D NumPy array given a vector
+``x``).  Results binned on the basis of a data field labeled ``'CCD'`` can be generated
+via the kwarg ``per_ccd_stat=True``.
+
+Histograms
+----------
+
+The :class:`HistogramSysTest <stile.sys_tests.HistogramSysTest>` systematics tests generate 
+histograms, optionally using optimized bin widths.  You can pass a one-dimensional dataset to
+display its histogram; a list of one-dimensional datasets to generate histograms on the same axes;
+a formatted array (see :doc:`data`) and a field to generate a histogram for that field; or a list of
+formatted arrays and a field or list of fields to generate multiple histograms on the same axes.
+
+To operate on a data vector:
+
+>>> histogram_sys_test_vector = stile.HistogramSysTest()
+
+To operate on a formatted array,
+
+>>> histogram_sys_test_field = stile.HistogramSysTest('g1')
+
+or pass the field at runtime:
+
+>>> results = histogram_sys_test_vector(data, field='g1')
+
+To operate on multiple data vectors:
+
+>>> results = histogram_sys_test_vector([data, data2])
+
+or on multiple formatted arrays:
+
+>>> results = histogram_sys_test_field([data, data2])
+
+or
+
+>>> results = histogram_sys_test_vector([data, data2], field='g1').
+
+To plot multiple fields from the same array, you'll need to pass the same array multiple times, due
+to implementation details:
+
+>>> results = histogram_sys_test_vector([data, data], field=['g1', 'g2'])
+
+or of course you can just access those fields yourself:
+
+>>> results = histogram_sys_test_vector([data['g1'], data['g2']]).
+
+By default, the :class:`HistogramSysTest <stile.sys_tests.HistogramSysTest>` makes a histogram with
+50 bins.  You probably want to choose different binning.  You can have Stile pick the binning for you
+based on two different algorithms:
+
+>>> scott_rule_hist = stile.HistogramSysTest(binning_style='scott')  # Scott's rule
+>>> freedman_rule_hist = stile.HistogramSysTest(binning_style='freedman')  # Freedman-Diaconis rule
```

Comment:
If it's not too annoying, can you say briefly what these rules are?  I'm not familiar with them, and I think if we can write a single sentence that prevents people running off to google, that would be helpful.